pkg/aggregator/aggregator.go

```diff
@@ -119,17 +119,17 @@ type SubObjectId struct {
 
 type aggregator struct {
 	options   options
-	clock     clock.Clock
-	linkers   linker.LinkerList
-	logger    logrus.FieldLogger
-	spanCache spancache.Cache
-	tracer    tracer.Tracer
-	metrics   metrics.Client
-
-	sendMetric               metrics.Metric
-	sinceEventMetric         metrics.Metric
-	lazySpanMetric           metrics.Metric
-	lazySpanRetryCountMetric metrics.Metric
+	Clock     clock.Clock
+	Linkers   linker.LinkerList
+	Logger    logrus.FieldLogger
+	SpanCache spancache.Cache
+	Tracer    tracer.Tracer
+	Metrics   metrics.Client
+
+	SendMetric               *metrics.Metric[*sendMetric]
+	SinceEventMetric         *metrics.Metric[*sinceEventMetric]
+	LazySpanMetric           *metrics.Metric[*lazySpanMetric]
+	LazySpanRetryCountMetric *metrics.Metric[*lazySpanRetryCountMetric]
 }
 
 func New(
```

Comment:
is this function not be used anymore?

Reply:
Right, maybe golangci didn't detect it because it is exported